featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1306,25 +1306,25 @@ def test_parallel_failure_raises_correct_error(pd_es):
 
 
 def test_warning_not_enough_chunks(pd_es, capsys, three_worker_scheduler):
```

Comment:
I believe you can put a single `# pragma: no cover` at the end of this line and avoid putting one on each line of the function body.

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/1829/commits/4492edc60cf14e1ef5e72a05a18b9ddebf3ce390

Reply:
Did this section get updated?  I'm seeing that the function still has a pragma no cover on each line.

Reply:
Sorry, fixed:
https://github.com/alteryx/featuretools/pull/1829/commits/a1ac50e2b4fdb5c767f474154d97e198c66e0294